server/controllers/patients/addHistoryLog/index.js

```diff
@@ -0,0 +1,62 @@
+const { boomify, historyLogSchema } = require('../../../utils');
+
+const {
+  addHistoryLogQuery,
+  getAppointmentsByIdQuery,
+  updateAppointmentStatusQuery,
+} = require('../../../database/queries');
+
+const addHistoryLog = async (req, res, next) => {
+  try {
+    const { appointmentId } = req.params;
+    const { description, price, payment } = req.body;
+
+    const isValid = await historyLogSchema.isValid({
```

Comment:
It's good practise to use historyLogSchema.validate(req.body,{abortEarly:false}) function, use can view the exact error of the input fields and send it back to the client

Reply:
Done